easybuild/toolchains/compiler/gcc.py

```diff
@@ -94,6 +94,7 @@ class Gcc(Compiler):
         # no support for -march on POWER; implies -mtune=native
         (systemtools.POWER, systemtools.POWER): '-mcpu=native',
         (systemtools.POWER, systemtools.POWER_LE): '-mcpu=native',
+        (systemtools.RISCV64, systemtools.RISCV): '-march=' + systemtools.get_isa_riscv(),  # use all extensions
```

Comment:
@julianmorillo How sure are we that the `isa` field in `/proc/cpuinfo` (which is what `get_isa_riscv` grabs) will always correspond to what GCC's `-march` option accepts as possible value?
Can we point to some GCC docs on this or something?

https://gcc.gnu.org/onlinedocs/gcc-15.2.0/gcc/RISC-V-Options.html mentions `-march=help`, maybe we should leverage that somehow (or at least mention it here)?